zio-http/src/main/scala/zhttp/service/server/ServerRequestHandler.scala

```diff
@@ -35,7 +35,7 @@ final case class ServerRequestHandler[R](
     decodeJRequest(jReq) match {
       case Left(err)  => cb(err.toResponse)
       case Right(req) =>
-        settings.http.execute(req).evaluate match {
+        settings.http.asHttp.execute(req).evaluate match {
```

Comment:
```suggestion
        settings.http.execute(req).evaluate match {
```

- [ ] add `execute` to `HttpApp`